docs/components/console/manage-plan/migrate-from-prof-to-starter.md

```diff
@@ -0,0 +1,49 @@
+---
+id: migrate-from-prof-to-starter.md
+title: Migrate from the Professional to the Starter plan
+description: "Want to migrate from Professional to Starter? Follow these steps."
+---
+
+:::note
+Here are a few important remarks to consider before completing the migration steps:
+
+- The migration is one way and cannot be reversed. Once you migrate to Starter, you cannot migrate back to Professional.
+- Since the two plans have different types of clusters included and fees for those, we recommend comparing the [Professional plan](https://camunda.com/blog/2023/05/camunda-professional-edition-accelerate-projects/) with the [Starter plan](https://camunda.com/blog/2023/09/camunda-starter/) to [understand your monthly costs](https://camunda.com/pricing/starter-plan-price-calculator/) before the migration.
+- General users and development/production cluster reservations in the Professional plan are migrated “as is” to the Starter plan, which may result in overage costs (e.g. production clusters in Professional will be transferred to production clusters in the Starter plan). If you are not using your production cluster in the Professional plan, we recommend you delete it beforehand and create a new development cluster in the Starter plan afterward.
+- Once you have edited the plan below, the changes will take effect on the first day of your next subscription period.
+- If you have any questions, do not hesitate to contact us at starter-plan@camunda.com.
+  :::
+
+<!--- Need to note moving forward that we have a specific contact for starter plan, or should this be added to the contact page? --->
+
+### Migrate from Professional to Starter plan
+
+To migrate to the Starter plan, take the following steps:
+
+1. From the Console, click **Organization > Billing > Manage Subscription.**
+
+<!--- I'm not seeing "Manage subscription" in ultrawombat, is this because it is a different enterprise setup? --->
```

Comment:
I can't see the UI based on https://console.cloud.ultrawombat.com/org/13e221f0-eae5-4fe2-a88f-c1fa5f17c5b3/management/billing, but assume this is because I'm in a different environment.

Reply:
@christinaausley could we update those images later as the changes seem minor?

Reply:
Yes! Works for me!